[docs] Improve grammar in 'Align list items' section (#27730)
mui-org/material-ui: docs/src/pages/components/lists/lists.md

## Align list items

> To change the list item alignment when displaying 3 lines or more, set the `alignItems` prop to "flex-start".

Reviewer: This changes the semantic otherwise

```suggestion
You should change the list item alignment when displaying 3 lines or more.
You can set the `alignItems="flex-start"` to do so.
```

## Align list items

> You should change the list item alignment when displaying 3 lines or more.

Reviewer 2: ```suggestion
You should change the list item alignment when displaying three lines or more.
```

Reviewer 2: To what, and why?

Reviewer: ```suggestion
The avatar is no longer correctly positioned when displaying three lines or more.
```

> The vertical alignment of the avatar is wrong when displaying three lines or more.

Reviewer 3: how does "wrong" look? "wrong" is an ambiguous term and oftentimes subjective. We should avoid "good" or "bad" or "right" or "wrong" in docs and just say explicitly what it refers to. For example
```suggestion
When displaying three lines or more, the avatar is not aligned at the top.
```

> You can set the `alignItems="flex-start"` to do so.

Reviewer 2: ```suggestion
Use `alignItems="flex-start"` to do so.
```

Reviewer: ```suggestion
You can set the `alignItems="flex-start"` to fix the positioning.
```

Reviewer 3: This is great feedback @mbrookes. I probably use "You can" instead of "Do" as well. This works as general advise similar to active vs. passive voice.

Though I would probably just merge these two sentences to avoid implicitness (and resolve https://github.com/mui-org/material-ui/pull/27730#discussion_r688817704 as well):

```suggestion
Use `alignItems="flex-start" when displaying three lines or more.`
```

But the "why" is still missing which also answers the question whether this is a SHOULD or MUST (in which case we may want to default to `flex-start` in the core implementation).

Reviewer: I gave the problem a try. See the latest commit for evaluation.

> You should set the `alignItems="flex-start"` prop to align the avatar at the top:

Reviewer: ```suggestion
You should set the `alignItems="flex-start"` prop to align the avatar at the top, following the Material Design guidelines:
```
